index.css

```diff
@@ -0,0 +1,419 @@
+/* Backwards-engineered from VS Code Webview (id="_defaultStyles"). Do not edit.  */
```

Comment:
Is it possible to .gitignore this file and have it "pulled" from VSCode's git repository as part of the build pipeline? (We play this trick in other places as well)

Reply:
This file is for development purposes (comments now added).

I think these styles are generated dynamically by VS Code. A quick look in their repo did not find this file.